docker/Dockerfile

```diff
@@ -146,8 +146,16 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # can be useful for both `dev` and `test`
 # explicitly set the list to avoid issues with torch 2.2
 # see https://github.com/pytorch/pytorch/pull/123243
-ARG torch_cuda_arch_list='7.0 7.5 8.0 8.9 9.0 10.0 12.0'
-ENV TORCH_CUDA_ARCH_LIST=${torch_cuda_arch_list}
+# Support both x86_64 and aarch64 architectures, with defaults for x86_64
+ARG torch_cuda_arch_list_x86_64='7.0 7.5 8.0 8.9 9.0 10.0 12.0'
+ARG torch_cuda_arch_list_arm64='8.0 8.7 8.9 9.0+PTX 12.1a'
```

Comment:
The list looks incorrect, could you confirm why 10.0 is removed for arm64? What is the source for the list?